index.html

```diff
@@ -55,17 +55,18 @@ <h1>Quiz - App</h1>
     </main>
 
     <footer class="footer">
```

Comment:
Please find a better naming for the CSS class or think of why and if you really need a class then!?